docs/docs/get_started/quickstart.mdx

```diff
@@ -417,6 +417,7 @@ Now that we have the tools, we can create an agent to use them. We will go over
 Install langchain hub first
 ```bash
 pip install langchainhub
+pip install langchain_openai
```

Comment:
lets not lump this in with langchainhub but include a specific instruction on why were installing it
also, if you are following this guide, theres a good chance youve already installed it

Reply:
sure we can add one more paragraph explaining why we need this

Reply:
updated